src/components/LocationDetails.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function LocationDetails(props) {
+  const { city, country, errorMessage } = props;
```

Comment:
You could destructure the props directly in the function statement and save yourself an extra line here:
```
function LocationDetails({ city, country, errorMessage }) {
```